src/test/java/com/wikia/webdriver/testcases/mercurytests/RedirectionTests.java

```diff
@@ -36,4 +43,49 @@ public void RedirectionTest_001_RedirectFromWWW() {
     new ArticlePageObject(driver).openWikiPage(url);
     Assertion.assertUrlEqualToCurrentUrl(driver, expectedUrl);
   }
+
+  @Test(groups = "RedirectionTest_002")
+     @Execute(onWikia = "mercuryautomationtesting")
+     public void RedirectionTest_002_RedirectFromFullSiteToOasis() {
+    MercuryFooterComponentObject footer = new MercuryFooterComponentObject(driver);
+    ArticleNavigationComponentObject navigation = new ArticleNavigationComponentObject(driver);
+    SkinHelper helper = new SkinHelper(driver);
+
+    new ArticlePageObject(driver).openWikiPage(url);
+    footer.clickFullSiteLink();
+    navigation.clickRandomArticle();
+
+    Assertion.assertTrue(helper.isSkin(Skin.OASIS));
+  }
+
+  @Test(groups = "RedirectionTest_003")
+  @Execute(onWikia = "mercuryautomationtesting", asUser = User.USER)
+  public void RedirectionTest_003_RedirectFromFullSiteToOasis() {
```

Comment:
Agreed, changed. Thanks for the input
AFABognaK this approach is also taken in many ads tests